controllers/articles.controller.js

```diff
@@ -50,5 +51,18 @@ function getAllArticlesData(_, response, next) {
         next(err)
     })
 }
+
+function patchArticleById(request, response, next) {
+    const {article_id} = request.params;
+    const {body} = request;
+    const promises = [selectArticle(article_id), updateArticle(article_id, body)];
+    Promise.all(promises)
+    .then((promisesResults) => {
+        response.status(200).send({updatedArticle: promisesResults[1]})
+    })
+    .catch(err => {
+        next(err)
+    })
+}
     
```

Comment:
A valid request on a non existent article should give a PSQL error removing the needs to check it exists.